test/e2e-upgrade-tests.sh

```diff
@@ -40,8 +40,8 @@ function install_latest_release() {
   local url="https://github.com/knative/serving/releases/download/${LATEST_SERVING_RELEASE_VERSION}"
   local yaml="serving.yaml"
 
-  # install serving core if installing for Gloo or Kourier
-  if [[ -n "${GLOO_VERSION}" || -n "${KOURIER_VERSION}" || -n "${AMBASSADOR_VERSION}" ]]; then
+  # serving.yaml contains the Istio integration, which we don't need for other ingress controllers.
+  if [[ -n "${GLOO_VERSION}" || -n "${KOURIER_VERSION}" || -n "${AMBASSADOR_VERSION}" || -n "${CONTOUR_VERSION}" ]]; then
```

Comment:
You could then just check if `INGRESS_CLASS` is empty of starts with `istio.`.

Reply:
I switched it to use serving-core by default, and look at `ISTIO_VERSION` to switch to serving.yaml

Reply:
Awesome!